Snake/BusinessLayer/GameSnake.cs

```diff
@@ -7,27 +7,30 @@
 namespace BusinessLayer
 {
     public class GameSnake
-    {
+    {
         enum Direction {right, down, left, up};
         Direction directionFacing;
         private int snakeLength;
         private List<Point> currentPosition;
 
         //Method to create new snake
```

Comment:
Comments in code should explain the "why" you are doing something that couldn't be read from reading the code e.g. Bad comment 
//returns a string 
public string GetString()
{
  returns string.empty;
}
from the code i can read this method returns a string.
Good comment:
//we needed a new method to add validation when this called from UI
public string GetStringWithValidation()
{
  string message = GetString();
  if (message.length > 1)
 { 
   throw new exception()
} else
........